nexus/tests/integration_tests/endpoints.rs

```diff
@@ -463,150 +489,216 @@ lazy_static! {
         // Global IAM policy
         VerifyEndpoint {
             url: *POLICY_URL,
-            visibility: Visibility::Public,
             allowed_methods: vec![
-                AllowedMethod::Get,
-                AllowedMethod::Put(
-                    serde_json::to_value(
-                        &shared::Policy::<authz::FleetRole> {
-                            role_assignments: vec![]
-                        }
-                    ).unwrap()
-                ),
+                AllowedMethod {
```

Comment:
In terms of reviewing these changes, since there are a lot of them and I want to make sure I got it right, I think the rule is: for any endpoint that was already in this file, whatever visibility it had before should now be applied identically and individually to each of its methods, modulo that Public becomes PublicPermissionRequired.  Is that right?